.github/ISSUE_TEMPLATE/monorepo-triage.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Access Voucher
+about: Request triage access to the Solana monorepo
+title: Request Access (Level 1) for [Username]
+labels: access
+assignees: joeaba
+
+---
+
+## Requesting Access Level 1 for [Username] to the Solana monorepo
+<!--
+  Access requests are defined in the Contributor Access Policy repository
+  It is up to you as the requester to gather the
+  appropriate number of vouchers from others.
+-->
+Discord ID:
```

Comment:
Let's get fancy and use a github forms [input](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#input) element to get the discord id.